tools/release/create-draft-release-notes.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: AGPL-3.0-only
+
+set -e
+
+# Load common lib.
+CURR_DIR="$(dirname "$0")"
+. "${CURR_DIR}/common.sh"
+
+find_last_release
+find_prev_release
+
+CHANGELOG_PATH="${CURR_DIR}/../../CHANGELOG.md"
+
+#
+# Contributors
+#
+
+NUM_PRS=$(git log --pretty=format:"%s" "${LAST_RELEASE_TAG}...${PREV_RELEASE_TAG}" | grep -Eo '#[0-9]+' | wc -l | grep -Eo '[0-9]+')
+NUM_AUTHORS=$(git log --pretty=format:"%an" "${LAST_RELEASE_TAG}...${PREV_RELEASE_TAG}" | sort | uniq -i | wc -l | grep -Eo '[0-9]+')
```

Comment:
Could you remind me of why you use the three-dot syntax here?

Reply:
Also, does it make any difference if you do `${PREV_RELEASE_TAG}...${LAST_RELEASE_TAG}` instead?